src/main/python/keymaps.py

```diff
@@ -24,6 +24,7 @@
     ("Swedish (QWERTY)", swedish.keymap),
     ("Swedish (SWERTY)", swedish_swerty.keymap),
     ("Swiss (QWERTZ)", swiss.keymap),
+    ("Ukrainian (ЙЦУКЕН)", ukrainian.keymap),
```

Comment:
it should go after Turkish to follow alphabet sorting

Reply:
fixed